write failures to error output

ketumbra opened the pull request:
Change failures from verbose to error output

#### Pull Request (PR) description
During application of a MOF, not all resources were created. Finding out why required searching through the verbose output. Errors should be output using the error stream so they are highlighted and easily available.

#### This Pull Request (PR) fixes the following issues
- None

ketumbra committed:
write failures to error output

Change failures from verbose to error output

NikCharlebois reviewed (changes requested):
Thanks, can you also add an entry in the Changelog.md file to describe the change?

ketumbra committed:
Update CHANGELOG.md

add description of changes to error output

ketumbra commented:
@microsoft-github-policy-service agree

ketumbra committed:
minor error msg tweak

Made failures during chang*ing* a policy use consistent tense as for errors when deleting and creating a policy

ketumbra commented:
Sure thing. Done.

NikCharlebois committed:
Update CHANGELOG.md
